docs/TASKS.md

```diff
@@ -1,11 +1,31 @@
 Tasks
 =====
 
-A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>.
+A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>. The manifest can be divided into two parts: Header and Workflow.
```

Comment:
I would say it is divided into two parts, not can be. It's a little confusing to me that the part directly below this isn't going into describing either the Header or the Workflow, maybe move this down below this section and add a header here about "Task states".